.github/workflows/pr.yml

```diff
@@ -96,3 +96,9 @@ jobs:
       uses: kittaakos/upload-artifact-as-is@v0
       with:
         path: ./**/*.apk
+
+    - name: Check for modified DB migrations
+      run: if git diff --name-only --diff-filter=DMR origin/master... | grep "^common/schemas"; then exit 1; fi
+
+    - name: Check for missing DB migrations
+      run: test -n "$(git status --porcelain)"
```

Comment:
This check isn't working as intended, see this test PR for example output: https://github.com/jpelgrom/home-assistant-android/pull/2. I think you need to use the output to exit with a code other than 0 to fail the step/workflow.

Reply:
Changed around the command, looks to be working now.

Reply:
Looks like I commented on the wrong line, changed files is now working but in the step named 'Check for _missing_ DB migrations', the original check for changed files still doesn't seem to work correctly